net-misc/streamlink/streamlink-9999.ebuild

```diff
@@ -22,15 +22,20 @@ RDEPEND="dev-python/pycryptodome[${PYTHON_USEDEP}]
 	virtual/python-futures[${PYTHON_USEDEP}]
 	virtual/python-singledispatch[${PYTHON_USEDEP}]
 	dev-python/backports-shutil_which[$(python_gen_usedep 'python2*')]
```

Comment:
@soredake, can you please clarify for me why didn't you change `python_gen_usedep` to `python_gen_cond_dep` also here?